eng/pipelines/templates/variables/image.yml

```diff
@@ -24,3 +24,6 @@ variables:
     value: windows
   - name: MACOS
     value: macOS
+
+  - name: LinuxImageDemand
+    value: ImageOverride -equals $(LINUXVMIMAGE)
```

Comment:
Did this double variable reference actually resolve?

Reply:
It does